examples/server/server.c

```diff
@@ -1840,6 +1958,69 @@ THREAD_RETURN WOLFSSL_THREAD server_test(void* args)
     if (ctx == NULL)
         err_sys_ex(catastrophic, "unable to get ctx");
 
+#ifdef WOLFSSL_WOLFSENTRY_HOOKS
```

Comment:
Please format the lines to avoid longer than 80 chars. At 1962, 1964, 1974-1978, 2012, 2020.

Reply:
:heavy_check_mark: